order.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/normalize.css" />
+    <link rel="stylesheet" href="./css/order.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Karma-order</title>
+  </head>
+  <body>
+    <header>
+      <div class="header">
+        <a class="header-logo" href="#">
+          <img
+            class="logo-image"
+            src="./img/karma-logo.svg"
+            alt="logo of karma"
+          />
+        </a>
+        <nav class="nav">
+          <!-- (B) THE HAMBURGER -->
+          <label for="hamburger">&#9776;</label>
+          <input type="checkbox" id="hamburger" />
+          <ul class="nav-menu">
+            <li class="list-item">
+              <a href="./index.html" class="nav-link nav-active">Meet Karma</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">How it Works</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Store</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            <li class="list-item">
+              <a href="#" class="nav-link">Help</a>
+            </li>
+            <li class="list-item">
+              <a href="./order.html" class="nav-link">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section class="form-section">
+        <div class="form-container">
+          <h1>Order your Karma wifi device today!</h1>
+          <form action="">
+            <div class="oneline-input">
+              <div class="borders">
```

Comment:
Could you re-name this something more generic and less tied to its style, for example `input-wrapper`?